querybook/webapp/components/EmbeddedDataDocPage/EmbeddedDataDocPage.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { FullHeight } from 'ui/FullHeight/FullHeight';
+import { DataDoc } from 'components/DataDoc/DataDoc';
+
+interface IProps {
+    id: number;
+}
+
+const EmbeddedDataDocPage: React.FunctionComponent<IProps> = ({ id }) => {
+    return (
+        <FullHeight flex={'column'} className="EmbeddedDataDocPage">
+            <div>
```

Comment:
this <div> is not necessary?

Reply:
This gives the embedded datadoc the ability to scroll. Setting overflow on the DataDoc component didn't give the desired effect

Reply:
> Just to confirm, div is needed for DataDoc scroll, right?
Yup, exactly